pypeit/core/extract.py

```diff
@@ -222,8 +219,6 @@ def extract_boxcar(sciimg, ivar, mask, waveimg, skyimg, box_radius, spec, base_v
         Wavelength image.
     skyimg : float `numpy.ndarray`_, shape (nspec, nspat)
         Image containing our model of the sky
-    box_radius : :obj:`float`
-        Size of boxcar window in floating point pixels in the spatial direction.
     spec : :class:`~pypeit.specobj.SpecObj`
```

Comment:
Can you add a note here in the docs indicating that the boxcar radius that is being used here should be set in the spec object.